src/features/comments/comment.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from "react";
+import { CommentsList } from "./commentsList";
+import {
+  Divider,
+  Box,
+  Typography,
+  IconButton,
+  List,
+  ListItemText,
+} from "@mui/material";
+import ForumRoundedIcon from "@mui/icons-material/ForumRounded";
+import NorthIcon from "@mui/icons-material/North";
+import SouthIcon from "@mui/icons-material/South";
+import ReplyIcon from "@mui/icons-material/Reply";
+import { relativeDays } from "../../components/helperFunctions";
+export const Comment = ({ comment }) => {
+  const replies = comment.data?.replies?.data?.children;
+  const date = new Date(comment.data.created_utc * 1000);
```

Comment:
This logic of doing * 1000 should be a helper, like `date_from_reddit_timestamp`